modules/openapi-generator/src/main/resources/swift5/api.mustache

```diff
@@ -10,15 +10,19 @@ import PromiseKit{{/usePromiseKit}}{{#useRxSwift}}
 import RxSwift{{/useRxSwift}}{{#useCombine}}
 #if canImport(Combine)
 import Combine
-#endif{{/useCombine}}{{#swiftUseApiNamespace}}
+#endif{{/useCombine}}{{#useVapor}}
+import Vapor{{/useVapor}}{{#swiftUseApiNamespace}}
 
-extension {{projectName}}API {
+extension {{projectName}} {
 {{/swiftUseApiNamespace}}
 
 {{#description}}
 /** {{description}} */{{/description}}
 {{#objcCompatible}}@objc {{/objcCompatible}}{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}open{{/nonPublicApi}} class {{classname}}{{#objcCompatible}} : NSObject{{/objcCompatible}} {
+    {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} init() { }
```

Comment:
Is there any advantage in using the instance method instead of calling a static method?
If this change is made, this should be surrounded with
```
{{#useVapor}}
...
{{/useVapor}}
```